chaoslib/litmus/disk_fill_by_litmus.yml

```diff
@@ -0,0 +1,161 @@
+- name: Get a Random Pod name
+  shell: >
+    kubectl get pods -l {{ label }} -n {{ namespace }} --no-headers -o custom-columns=:metadata.name | shuf -n 1 
+  args:
+    executable: /bin/bash
+  register: pod_name
+
+- name: Get ephemeral-storage Limit Value
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[?(@.name=="{{ app_container }}")].resources.limits.ephemeral-storage}' | grep -Eo '+|[0-9]+'
+  args:
+    executable: /bin/bash
+  register: limit_value
+
+- name: Get ephemeral-storage Limit storage unit
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[?(@.name=="{{ app_container }}")].resources.limits.ephemeral-storage}' | grep -Eo '[[:alpha:]]+'
+  args:
+    executable: /bin/bash
+  register: limit_value_storage
+
+- name: Get ephemeral-storage Request Value
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[?(@.name=="{{ app_container }}")].resources.requests.ephemeral-storage}' | grep -Eo '+|[0-9]+'
+  args:
+    executable: /bin/bash
+  register: req_value
+
+- name: Get ephemeral-storage Request storage unit
+  shell: >
+    kubectl get pods {{ pod_name.stdout }} -n {{ namespace }} -o jsonpath='{.spec.containers[?(@.name=="{{ app_container }}")].resources.requests.ephemeral-storage}' | grep -Eo '[[:alpha:]]+'
+  args:
+    executable: /bin/bash
+  register: req_value_storage
+
+- name:
+  include_vars: /experiments/generic/disk_fill/test_vars_diskfill.yml
+
+- name: Fetch the value of Request Storage Unit in KB
+  set_fact:
+    req_storage_KB: "{{ lookup('vars', req_value_storage.stdout) }}"
+
+- name: Fetch the value of Limit Storage Unit in KB
+  set_fact:
+    limit_storage_KB: "{{ lookup('vars', limit_value_storage.stdout)}}"
+
+- name: Multiply the request value with unit in KB
+  set_fact:
+    req_KB: "{{ req_value.stdout|int * req_storage_KB|int }}"
+
+- name: Multiply the limit value with unit in KB
+  set_fact:
+    limit_KB: "{{ limit_value.stdout|int * limit_storage_KB|int }}"
+
+- name: Convert the fill percent
+  set_fact: 
+    fill: "{{ fill_percent |float / 100 }}"
+
+- name: Fill percent into storage
+  set_fact:
+    to_fill: "{{ fill|float * limit_KB|float }}"
```

Comment:
Can this be a separate util under `chaoslib/litmus/disk_fill/<>`, to keep it more modular? The main disk_fill chaoslib may "own" separate utils performing a very specific action - say, in this case the action is "convert disk fill percentage to storage"